addons/sale_stock/report/report_stock_forecasted.py

```diff
@@ -17,6 +17,7 @@ def _compute_draft_quantity_count(self, product_template_ids, product_variant_id
             quantities = so_lines.mapped(lambda line: line.product_uom._compute_quantity(line.product_uom_qty, product_uom))
             out_sum = sum(quantities)
         res['draft_sale_qty'] = out_sum
+        res['draft_sale_orders'] = so_lines.mapped("order_id").sorted(key=lambda r: r.name)
```

Comment:
detail but for the next lambda try to get a variable that represent the model your are browsing. e.g here it's sale order, thus I would choose lambda so: so.name